### PR TITLE
chore: update jimmctl command info

### PR DESCRIPTION
**Pull request by kian99**
## Description
As part of generating automatic documentation for `jimmctl` commands, this PR fixes some issues that were blocking the way toward automatic doc generation. The fixes here are also useful to the jimmctl->Juju feature since these commands will either be copied to Juju or moved into a plugin (except for any that get removed).

The full changes and reasons:
1. Populate the `Examples` field of all commands for correct documentation generation.
2. Changed examples that contained <controller> to an actual string like mycontroller. I copied Juju's help text e.g. `juju add-cloud -h` uses mycontroller and mycloud in their examples.
3. Populate the `Args` field of all commands for correct documentation generation.
4. Removed indentation from the command's doc string to match Juju's output.
5. Ensure example all have consistent indentation with 4 spaces instead of a tab to match Juju.
6. Nested commands like auth should have the `UsagePrefix` field set.

Partially fixes [JUJU-7172](https://warthogs.atlassian.net/browse/JUJU-7172)

## Engineering checklist
*Check only items that apply*

- [x] Documentation updated
- [ ] Covered by unit tests
- [ ] Covered by integration tests

[JUJU-7172]: https://warthogs.atlassian.net/browse/JUJU-7172?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: cmd/jimmctl/cmd/group.go [hunk 1]
```diff
@@ -20,47 +20,46 @@ import (
...
 `
 	renameGroupDoc = `
-rename command renames a group in jimm.
-
-Example:
-	jimmctl auth group rename <name> <new name>
+The rename command renames a group in jimm.
+`
+	renameGroupExample = `
+    jimmctl auth group rename mygroup newgroup
```
Comment (SimoneDutto) [inline cmd/jimmctl/cmd/group.go:38]: i know it's for consistency with juju, but <newname> is clearer.

**Review by SimoneDutto — APPROVED** [review]
nice work

File: cmd/jimmctl/cmd/auth.go [hunk 2]
```diff
@@ -6,15 +6,16 @@ import (
 	jujucmd "github.com/juju/cmd/v3"
 )
 
-var authDoc = `
-auth enables users to manage authorisation model used by JIMM.
+const authDoc = `
+The auth command enables users to manage authorisation model used by JIMM.
```
Comment (alesstimec) [inline cmd/jimmctl/cmd/auth.go]: this is/was a bit vague/misleading. We manage permissions/tuples, while the authorisation model is fixed (created by the charm).

**Review by alesstimec — APPROVED** [review]
LGTM with a suggestion

File: cmd/jimmctl/cmd/auth.go (hunk 2 above)
Comment (kian99) [reply cmd/jimmctl/cmd/auth.go]: I'll update it.

**Pull request merged**